Source/Gral/GRALShapeFiles/SHPLine.cs

```diff
@@ -24,5 +24,9 @@ public class SHPLine
         public int NumPoints;
         public int[] Parts;
         public GralDomain.PointD[] Points;
+        /// <summary>
+        /// Z value of a Point
+        /// </summary>
+        public double[] PointsZ;
```

Comment:
The shape class got an optional Z array, because the height is not needed in all cases.